static/index.html

```diff
@@ -1,17 +1,75 @@
 <!DOCTYPE html>
 <html>
-    <head>
-        <meta charset="UTF-8">
-        <title>KaTeX Test</title>
-        <script defer src="/main.js" type="text/javascript"></script>
-    </head>
-    <body>
-        <textarea id="input" rows="5">
+<!-- referenced from issue #1615 -->
```

Comment:
It would be better to refer to `website/pages/index.html` in this comment. And maybe it should move down 

You also need to consistently use either indentation of 4 (as before in this file) or 2 (as in `website/pages/index.html`), and `<head>` and `<body>` should be indented within `<html>`.